src/view-components/tabbed-view/tabbed-view.css

```diff
@@ -59,3 +59,11 @@
 .summariesnav a {
   color: inherit;
 }
+
+.routesContainer {
+  margin: 21px;
```

Comment:
Why not 22px? Or 20px? Can we have this in `em` or `rem`? Any reason for this particular number? (Just curious)

Reply:
the `21px` is the default margin between widgets not sure the reason for it but my guess is from the Figma designs.

Am converting from `px` to `rem`